onnxruntime/core/providers/webnn/builders/impl/binary_op_builder.cc

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include <core/providers/common.h>
+
+#include "core/providers/webnn/builders/model_builder.h"
+#include "core/providers/webnn/builders/op_builder_factory.h"
+#include "core/providers/webnn/builders/helper.h"
+
+#include "base_op_builder.h"
+
+namespace onnxruntime {
+namespace webnn {
+
+class BinaryOpBuilder : public BaseOpBuilder {
+  // Add operator related
+ private:
+  Status AddToModelBuilderImpl(ModelBuilder& model_builder, const Node& node,
+                               const logging::Logger& logger) const override ORT_MUST_USE_RESULT;
+
+  // Operator support related
+ private:
+  int GetMinSupportedOpSet(const Node& node) const override;
+};
+
+// Add operator related
+
+Status BinaryOpBuilder::AddToModelBuilderImpl(ModelBuilder& model_builder, const Node& node,
+                                              const logging::Logger& /* logger */) const {
+  const auto& op_type(node.OpType());
+
+  emscripten::val input0 = model_builder.GetOperand(node.InputDefs()[0]->Name());
+  emscripten::val input1 = model_builder.GetOperand(node.InputDefs()[1]->Name());
+  emscripten::val output = emscripten::val::object();
+  if (op_type == "Add") {
+    output = model_builder.GetBuilder().call<emscripten::val>("add", input0, input1);
+  } else {
```

Comment:
We probably could support other binary ops like `sub`, `mul`, `div`, `min` and `max`.

Reply:
Indeed.

Reply:
@huningxin, in ONNX, `min`, `max` accept multiple inputs, we should implement them in a separate PR.